src/Components/Reviews.jsx

```diff
@@ -1,27 +1,39 @@
-import { getReviews } from "../api"
+import { getReviews, getReviewsByCategory } from "../api"
 import { useEffect, useState } from 'react'
-import { Link } from "react-router-dom"
+import { Link, useSearchParams } from "react-router-dom"
+
 
 const Reviews = () => {
     const [reviewList, setReviews] = useState([])
     const [isLoading, setIsLoading] = useState(true)
+    const [searchParams, setSearchParams] = useSearchParams()
+    const sortByQuery= searchParams.get('category')
```

Comment:
I don't think sortByQuery is the best name for this variable. We aren't sorting anything!